platform.go

```diff
@@ -64,9 +61,7 @@ func (p *platformPlugin) handleClipboardSetData(arguments interface{}) (reply in
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to decode json arguments for handleClipboardSetData")
 	}
-	p.glfwTasker.Do(func() {
-		p.window.SetClipboardString(newClipboard.Text)
-	})
+	p.window.SetClipboardString(newClipboard.Text)
```

Comment:
So, just make sure I get this right; this code is guaranteed to run in the glfw main thread, which is why we don't need to add this as task on the glfwTasker?

Reply:
Yes, `HandleFuncSync` runs on the main thread.